cypress/integration/article.js

```diff
@@ -122,6 +122,11 @@ describe('Sample Article Page', () => {
             'property="og:url"',
             'http://developer-test.mongodb.com/article/3-things-to-know-switch-from-sql-mongodb'
         );
+        cy.get('link[rel="canonical"]').should(
+            'have.prop',
+            'href',
+            `${PROD_ARTICLE_URL}/`
```

Comment:
How do we know the PROD_ARTICLE_URL here doesn't have a trailing slash already?

Reply:
This is actually a CI test test file and `PROD_ARTICLE_URL` is defined above as `https://developer.mongodb.com${ARTICLE_WITH_SERIES_URL}`